src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -192,15 +196,11 @@ export const NavSearchBar: React.FC<NavSearchBarProps> = ({ venueId }) => {
           search results
         </div>
 
-        {isLoading || isEventsLoading ? (
-          <Loading />
-        ) : (
-          <div className="NavSearchBar__search-results">
-            {foundRooms}
-            {foundEvents}
-            {foundUsers}
-          </div>
-        )}
+        <div className="NavSearchBar__search-results">
+          {isRoomsLoading ? <Loading /> : foundRooms}
+          {isEventsLoading ? <Loading /> : foundEvents}
+          {foundUsers}
+        </div>
```

Comment:
Forgot I had changed this. I don't think it will add to performance. Originally only displayed one spinner until both rooms and events were available results. Now it displays both spinners, then one, then none. Should probably revert it to the original.

Reply:
Reverted in bdb277f6eb41e3a42e9b3b1383c20c03e2e59adc